spec/rubocop/cop/rspec/repeated_example_group_body_spec.rb

```diff
@@ -74,6 +74,70 @@
     RUBY
   end
 
+  it 'does not register offense if module arg is different' do
+    expect_no_offenses(<<-RUBY)
+      describe Array do
+        it { described_class.cool_predicate_method }
+      end
+
+      describe Hash do
+        it { described_class.cool_predicate_method }
+      end
+    RUBY
+  end
+
+  it 'registers an offense for repeated context body and module arg' do
+    expect_offense(<<-RUBY)
+      context Array do
+      ^^^^^^^^^^^^^^^^ Repeated context block body on line(s) [5]
+        it { described_class.cool_predicate_method }
+      end
+
+      context Array do
+      ^^^^^^^^^^^^^^^^ Repeated context block body on line(s) [1]
+        it { described_class.cool_predicate_method }
+      end
+    RUBY
+  end
+
+  it 'does not register offense if namespaced module arg is different' do
```

Comment:
It might sound reckless, but the following examples look excessive to me.

Reply:
yeah, i needed this when tdd-ing, because my initial solution didn't cover it, but i can probably reduce the tests down

Reply:
now there are just 3 tests: allow different module, allow different namespace for module, complain when identical body and module. all with real looking expectations and classes from stdlib